src/Compilers/CSharp/Portable/Binder/LockBinder.cs

```diff
@@ -15,6 +13,10 @@ namespace Microsoft.CodeAnalysis.CSharp
 {
     internal sealed class LockBinder : LockOrUsingBinder
     {
+        private const string LockTypeFullName = "System.Threading.Lock";
```

Comment:
>LockTypeFullName [](http://example.com/codeflow?start=29&length=16)

It looks like this const can be local to ```TryFindLockTypeInfo```

Reply:
And the other two consts as well. In addition to "polluting" the lookup space, const fields take more space in the resulting assembly image.